src/web/app/page.component.scss

```diff
@@ -85,12 +85,17 @@ nav {
   }
 
   .main-content {
-    padding: 0 0;
+    padding: 10px 15px;
   }
 
   .footer {
     padding: 5px 0;
   }
+
+  .footer .row {
+    margin-left: 0;
+    margin-right: 0;
+  }
```

Comment:
Similar to `.main-content`, this can be removed if we change the padding of `.footer` to `5px 15px`.